lib/molinillo/resolution.rb

```diff
@@ -195,10 +219,8 @@ def unwind_for_conflict
       # @return [Integer] The index to which the resolution should unwind in the
       #   case of conflict.
       def state_index_for_unwind
-        current_requirement = requirement
-        existing_requirement = requirement_for_existing_name(name)
         index = -1
-        [current_requirement, existing_requirement].each do |r|
+        activated.vertex_named(name).requirements.each do |r|
```

Comment:
I'm wondering about the implications of the ordering with just doing `Vertex#requirements`, as opposed to the "smart" buckjumping -- can you explain this change?